python/dlisio/plumbing/basicobject.py

```diff
@@ -16,6 +16,21 @@ class BasicObject():
     """
 
     attributes = {}
+    """dict: Maps dlis-file attributes to class attributes.
```

Comment:
Add something like this to clearly state the intent of attribute: "Index of what attributes this object type is expected to have. dlisio will use this index when parsing object. If an attribute is not in this index, it will be ignored when parsing the file"?